.prettierignore

```diff
@@ -19,3 +19,9 @@
 
 # These are auto-generated by css-modules-typescript-loader
 **/*.scss.d.ts
+
+!wwwroot
+!wwwroot/test
+!wwwroot/test/Magda
+!wwwroot/test/Magda/shareKeys/**/
+!wwwroot/test/Magda/shareKeys/**/*.json
```

Comment:
Might as well put prettier on new things I'm adding. You have to unignore each folder along the path, but this will only apply to any JSON files matching `wwwroot/test/Magda/shareKeys/**/*.json`